spring-cloud-app-broker-acceptance-tests/src/test/java/org/springframework/cloud/appbroker/acceptance/CloudFoundryAcceptanceTest.java

```diff
@@ -136,6 +141,7 @@ private String brokerClientId() {
 	void setUp(TestInfo testInfo, BrokerProperties brokerProperties) {
 		List<String> appBrokerProperties = getAppBrokerProperties(brokerProperties);
 		blockingSubscribe(initializeBroker(appBrokerProperties));
+		blockingSubscribe(initializeClient());
```

Comment:
We should be explicit which client this is, since we have two of them and it's not immediately obvious what's going on here. Ideally I think we'd use a user here instead of a client.